src/main/frontend/src/components/entrypoints/users/Notifications.vue

```diff
@@ -159,8 +159,8 @@ export default {
         };
     },
     mounted() {
-        const invites = $('.invite-content');
-        invites.css('height', invites.width());
+        const invites = document.querySelector('.invite-content');
+        invites.style.height = invites.style.width;
```

Comment:
this is **not** the same thing. The element has no width. The jquery width method calculates the actual width of the element. What you changed it to is what I originally had, but it's not the same, and won't work.

Reply:
Thats from before testing. Will work our another way of doing this WITHOUT JQUERY

Reply:
It is .offsetWidth for reference.